### PR TITLE
(case 20748) Add material inspector to Luci.js

### PR DESCRIPTION
**Pull request by sabrina-shanman**
https://highfidelity.manuscript.com/f/cases/20748/Add-ability-to-inspect-material-properties-to-luci-js

This PR adds a button to luci.js labeled "Material" that allows you to see the material properties for a mesh part on an object. It will help us fix some rendering/model-related bugs faster.

## TEST PLAN
- This PR touches a developer tool not normally accessible to the end-user. I recommend whichever developer reviews this PR to run the updated luci.js and related files and verify the script still works, and try using the new Material Inspector by clicking on the "Material" button.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16577/HighFidelity-Beta-PR14745-556a2ec.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16577/HighFidelity-Beta-Interface-PR14745-556a2ec.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16577/HighFidelity-Beta-PR14745-556a2ec.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16577/HighFidelity-Beta-Interface-PR14745-556a2ec.exe

**Comment by sabrina-shanman** [comment]
Build this please.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16592/HighFidelity-Beta-PR14745-556a2ec.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16592/HighFidelity-Beta-Interface-PR14745-556a2ec.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16592/HighFidelity-Beta-PR14745-556a2ec.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16592/HighFidelity-Beta-Interface-PR14745-556a2ec.exe

**Comment by sabrina-shanman** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16601/HighFidelity-Beta-PR14745-556a2ec.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16601/HighFidelity-Beta-Interface-PR14745-556a2ec.exe

**Comment by sabrina-shanman** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16715/HighFidelity-Beta-PR14745-556a2ec.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16715/HighFidelity-Beta-PR14745-556a2ec.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16715/HighFidelity-Beta-Interface-PR14745-556a2ec.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16715/HighFidelity-Beta-PR14745-556a2ec.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16715/HighFidelity-Beta-Interface-PR14745-556a2ec.exe

File: scripts/developer/utilities/render/deferredLighting.qml [hunk 1]
```diff
@@ -303,5 +302,13 @@ Rectangle {
                 }
             }
         }
+        Row {
+            HifiControls.Button {
+                text: "Material"
+                onClicked: {
+                sendToScript({method: "openMaterialInspectorView"});
```
Comment (AndrewMeadows) [inline scripts/developer/utilities/render/deferredLighting.qml]: Bad indentation.

**Comment by qlimnavarro** [comment]
Currently Testing

**Pull request merged**